docs/api.md

```diff
@@ -209,6 +209,8 @@ Cancels an open order by hash.
 Please note you will need to provide a valid cancelation signature in the Authorization header in order to cancel an order.
 The Authorization header signature should be hashed according to [EIP712](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-712.md) and include the original orderHash but will not include any information about the order format, version, or chainId since these are already baked-into the hash of the order. You can see working examples of signing in the [LimitOrders](https://github.com/dydxprotocol/solo/blob/master/src/modules/LimitOrders.ts) module of Solo.js.
 
+The response will have a status of 200 as long as the order already existed and the signature is valid (even if the order is already unfillable for any reason). For example, if a user cancels an order twice, then 200 will be returned both times. In addition, canceling a filled order will return 200 but will NOT update the status of the order from `FILLED` to `CANCELED`.
```

Comment:
Add something about how 200 does not necessarily mean the thing was canceled, just that we received the request

Reply:
also the last sentence is confusing